### PR TITLE
Add Owlet60 Keyboard to qmk_firmware/keyboards/handwired

### PR DESCRIPTION
**Pull request by worthlessowl**
## Description

Add Owlet60 keyboard to keyboards/handwired. Keyboard is currently on a local groupbuy/preorder, I'm adding this to QMK so owner of this keyboard can easily make their firmware on QMKConfigurator. No changes to any of the QMK working, only adding my keyboard firmware. Decides to put on handwired because this will be a one-off project

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
Thanks for the contribution! :)

It looks like there are some merge conflict artifacts here (`keyboards/copenhagen_click/click_pad_v1/config.h`, `keyboards/vn66/rules.mk`), and you appear to have added the same keyboard in two different places (`keyboards/handwired/owlet60` and `keyboards/handwired/worthlessowl/owlet60`). Could you please fix these?

File: keyboards/copenhagen_click/click_pad_v1/config.h [hunk 1]
```diff
@@ -18,6 +18,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#include "pro_micro.h"
```
Comment (drashna) [inline keyboards/copenhagen_click/click_pad_v1/config.h]: Could these changes be pulled out and put into a different PR?

File: keyboards/vn66/rules.mk [hunk 2]
```diff
@@ -34,12 +34,21 @@ COMMAND_ENABLE = yes        # Commands for debug and configuration
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = no            # USB Nkey Rollover
+<<<<<<< HEAD:keyboards/vn66/rules.mk
```
Comment (drashna) [inline keyboards/vn66/rules.mk]: You have some merge artifacts here. 

You should be able to run `git checkout upstream/master keyboards/vn66/rules.mk` to fix this.

File: keyboards/handwired/owlet60/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# owlet60
+
+Owlet60, a TGR Alice inspired 60%-65% keyboard. Powered with a Pro Micro and 4051 mux multiplexed matrix. PCB can be permanently snapped off for a 60% layout.
+
+Keyboard Maintainer: [worthlessowl](https://github.com/worthlessowl)  
+Hardware Supported: Owlet60 PCB, standard Pro Micro
```
Comment (fauxpark) [inline keyboards/handwired/owlet60/readme.md]:
```suggestion
Hardware Supported: Owlet60 PCB, standard Pro Micro  
```

File: keyboards/copenhagen_click/click_pad_v1/config.h (hunk 1 above)
Comment (worthlessowl) [reply keyboards/copenhagen_click/click_pad_v1/config.h]: Hi drashna, thank you for reviewing my PR. I actually didn't intend to edit any other boards other than mine, but somehow some merge artifact got into my project, and I'm pretty confused on how to resolve it. Should i checkout this board just like how you show it below? Thank you, I'm sorry for this inconvenience.

About the custom matrix, yes, how the matrix is wired up is the reason. 

Im using a multiplexer on my matrix. It was inspired by Meira and Ronin, but i wired it slightly different. I use Col2Row configuration and connected all the columns to the multiplexer. So instead of digitally switching between column to read them, I use the multiplexer to analogously switch between columns.

Comment (drashna) [reply keyboards/copenhagen_click/click_pad_v1/config.h]: Okay, not a problem, stuff happens! 

For this, the simplest mething should be to just run:
```
git checkout upstream/master keyboards/copenhagen_click/click_pad_v1/config.h
```

That should pull the correct from from upstream (which should hopefully be set, if not, then `git remote add upstream https://github.com/qmk/qmk_firmware.git` followed by a `git fetch -all` should fix that)

File: keyboards/handwired/owlet60/matrix.c [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+    for (uint8_t i=0; i < MATRIX_ROWS; i++) {
+        raw_matrix[i] = 0;
+        matrix[i] = 0;
+    }
+
+    debounce_init(MATRIX_ROWS);
+
+    matrix_init_quantum();
+
+    DDRD &= ~(1<<5);
```
Comment (fauxpark) [inline keyboards/handwired/owlet60/matrix.c]:
```suggestion
    setPinInput(D5);
```

File: keyboards/handwired/owlet60/matrix.c [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+   PORTD &= ~(1<<5);
```
Comment (fauxpark) [inline keyboards/handwired/owlet60/matrix.c]:
```suggestion
   writePinLow(D5);
```

File: keyboards/handwired/owlet60/matrix.c [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+
+   DDRB &= ~(1<<0);
```
Comment (fauxpark) [inline keyboards/handwired/owlet60/matrix.c]:
```suggestion
   setPinInput(B0);
```

File: keyboards/handwired/owlet60/matrix.c [hunk 7]
```diff
@@ -0,0 +1,323 @@
...
+   PORTB &= ~(1<<0);
```
Comment (fauxpark) [inline keyboards/handwired/owlet60/matrix.c]:
```suggestion
   writePinLow(B0);
```

File: keyboards/copenhagen_click/click_pad_v1/config.h (hunk 1 above)
Comment (worthlessowl) [reply keyboards/copenhagen_click/click_pad_v1/config.h]: Thank you for your help! I've checked out the merge artifacts in problem. I hope everything is alright now.

File: keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk [hunk 8]
```diff
@@ -0,0 +1,48 @@
+SRC += matrix.c
```
Comment (drashna) [inline keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk]: You don't need all of this here.  

In fact, the only thing you probably need in this file is:

```
OLED_DRIVER_ENABLE = yes
```

File: keyboards/handwired/owlet60/matrix.c [hunk 9]
```diff
@@ -0,0 +1,323 @@
...
+
+  matrix_scan_quantum();
+  return (uint8_t)changed;
+}
+*/
+
+static void select_col_manual(uint8_t col) {
+    // emergency code, just a simple switch statement
+    
+    if(col == 0)
```
Comment (yanfali) [inline keyboards/handwired/owlet60/matrix.c]: for readability I would use a switch instead of if then else. It would also probably be more readable to just use an 2d array of 0, 1; set pin to low for 0 and set pin to high for 1; there are only 3 columns so, that's a 3 x 8 array, which isn't too bad. I think it would end up a lot more readable and less repetitive.

File: keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk (hunk 8 above)
Comment (worthlessowl) [reply keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk]: Thank you for pointing that out. I've deleted all the redundant rules on my latest commit.

File: keyboards/handwired/owlet60/matrix.c (hunk 9 above)
Comment (worthlessowl) [reply keyboards/handwired/owlet60/matrix.c]: Hello yanfali. Thank you for your suggestion. I've implemented a new select code with your suggestion in mind. If you think there's anything that I could improve upon, please tell me.

File: keyboards/handwired/owlet60/matrix.c (hunk 7 above)
Comment (worthlessowl) [reply keyboards/handwired/owlet60/matrix.c]: Thank you for your change request. I referenced an old firmware that didn't use the newer controller agnostic functions. I didn't know, back then when I was building this firmware, that they have an equivalent function in a much more elegant form.

File: keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk (hunk 8 above)
Comment (worthlessowl) [reply keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk]: Is there anything else i need to fix with my code? Thank you
